packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,197 @@
+      from: fundedAddress, to: burnAddress
+    })
+
+    // utxostoaccount burn
+    await container.call('utxostoaccount', [{ [burnAddress]: '1@0' }])
+    await container.generate(1)
+
+    // send to burn address with accountToAccount
+    await accountToAccount(container, '0', 1, {
+      from: fundedAddress, to: burnAddress
+    })
+
+    // send to burn address with accounttoutxos
+    const payload: BalanceTransferPayload = {}
+    payload[burnAddress] = '2@0'
+    await client.account.accountToUtxos(fundedAddress, payload)
+    await container.generate(1)
+
+    // send utxo to burn address
+    await client.wallet.sendToAddress(burnAddress, 10)
+    await container.generate(1)
+  }
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listBurnHistory', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history.length).toStrictEqual(7)
+    for (let i = 0; i < history.length; i += 1) {
+      const burnHistory = history[i]
+      expect(typeof burnHistory.owner).toStrictEqual('string')
+      expect(typeof burnHistory.blockHeight).toStrictEqual('number')
+      expect(typeof burnHistory.blockHash).toStrictEqual('string')
+      expect(typeof burnHistory.blockTime).toStrictEqual('number')
+      expect(typeof burnHistory.type).toStrictEqual('string')
+      expect(typeof burnHistory.txn).toStrictEqual('number')
+      expect(typeof burnHistory.txid).toStrictEqual('string')
+      expect(burnHistory.amounts.length).toBeGreaterThan(0)
+      expect(typeof burnHistory.amounts[0]).toStrictEqual('string')
+    }
+  })
+
+  it('first transaction should be masternode creation fee burn', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[6].owner.slice(0, 16)).toStrictEqual('6a1a446654784301')
+    expect(history[6].type).toStrictEqual('CreateMasternode')
+    expect(history[6].txn).toStrictEqual(2)
+    expect(history[6].amounts.length).toStrictEqual(1)
+    expect(history[6].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('second transaction should be after burn token', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[5].owner).toStrictEqual('6a19446654785404474f4c4404474f4c4408000000000000000007')
+    expect(history[5].type).toStrictEqual('CreateToken')
+    expect(history[5].txn).toStrictEqual(1)
+    expect(history[5].amounts.length).toStrictEqual(1)
+    expect(history[5].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('third transaction should be send token to burn address', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[4].owner).toStrictEqual(burnAddress)
+    expect(history[4].type).toStrictEqual('AccountToAccount')
+    expect(history[4].txn).toStrictEqual(1)
+    expect(history[4].amounts.length).toStrictEqual(1)
+    expect(history[4].amounts[0]).toStrictEqual('100.00000000@GOLD')
```

Comment:
```suggestion
    expect(history[4]).toStrictEqual({
      owner: burnAddress,
      type: 'AccountToAccount',
      txn: 1,
      amounts: ['100.00000000@GOLD'],
    })
```

Might be better to write it like this